pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -358,6 +326,14 @@ func (c *Controller) GetService(hostname host.Name) (*model.Service, error) {
 	return c.servicesMap[hostname], nil
 }
 
+func GetLocalityFromTopology(topology map[string]string) string {
+	locality := topology[NodeZoneLabelGA]
+	if _, f := topology[NodeRegionLabelGA]; f {
+		locality += "/" + topology[NodeRegionLabelGA]
```

Comment:
it is opposite,  region/zone

Reply:
Thanks! will add unit tests

Reply:
Done, also added the istio subzone label